source/engineering/OpenFOAM/single-node.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "Single-Node Jobs"
+---
+
+Small OpenFOAM jobs can be run on a single node.
+
+## Job Script Template
+
+Here is an example job submission script for a single-node OpenFOAM job:
+
+```{code-block} bash
+---
+linenos: true
+---
+#!/bin/bash
+## Submission script for Cluster
+#SBATCH --job-name=my_job
+#SBATCH --time=0-01:00:00 # hh:mm:ss
+#SBATCH --nodes=1
+#SBATCH --ntasks-per-node=1
+#SBATCH --cpus-per-task=1
+
+
+#SBATCH --partition=cpu
+
+## system error message output file
+## leave %j as it's being replaced by JOB ID number
+#SBATCH -e aai_LD_%j.err
```

Comment:
What does `aai_LD` in the filename mean here? Maybe it would be simpler to rename to `my_job_%j.err` and `my_job_%j.out`?
```suggestion
#SBATCH -e my_job_%j.err
```